batchalign/pipelines/avqi/engine.py

```diff
@@ -0,0 +1,262 @@
+"""
+AVQI Engine for Batchalign2
+Acoustic Voice Quality Index calculation for voice quality assessment
+"""
+
+import parselmouth
+import numpy as np
+from parselmouth.praat import call
+import re
+from typing import Tuple, Dict, Optional
+import os
+from pathlib import Path
+import logging
+
+from batchalign.pipelines.base import BatchalignEngine
+
+L = logging.getLogger('batchalign')
+
+
+class AVQIEngine(BatchalignEngine):
+    """Engine for calculating Acoustic Voice Quality Index (AVQI)."""
+    
+    def __init__(self):
+        super().__init__()
+        self._tasks = ["avqi"]
```

Comment:
this is not registered as a task; this is important because the order of tasks determine how pipelines are sequenced (for instance, user may want to run AVQI + morphotag); feel free to do one of:

1. register the task at https://github.com/TalkBank/batchalign2/blob/3d157eb11eb8e9d4381e80bc198d984da7bb7c29/batchalign/document.py#L23-L64
2. use `Task.Analysis` (I have a slight preference for this)